src/Bundle/Grid/Renderer/TwigBulkActionGridRenderer.php

```diff
@@ -50,7 +42,7 @@ public function renderBulkAction(GridViewInterface $gridView, Action $bulkAction
 
         $options = $this->optionsParser->parseOptions(
             $bulkAction->getOptions(),
-            $gridView->getRequestConfiguration()->getRequest(),
+            $request ?? new Request(),
```

Comment:
Just an idea since it's the only place where requestStack is used.

```suggestion
            $this->requestStack->getCurrentRequest() ?? new Request(),
```